static/js/latest-news.js

```diff
@@ -0,0 +1,115 @@
+function formatDate(date) {
+  const parsedDate = new Date(date);
+  const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  ];
+
+  const day = parsedDate.getDate();
+  const monthIndex = parsedDate.getMonth();
+  const year = parsedDate.getFullYear();
+
+  return day + " " + monthNames[monthIndex] + " " + year;
+}
+
+function htmlForLatestArticles(articles) {
+  const articlesTree = document.createDocumentFragment();
+  for (let i = 0; i < articles.length; i++) {
+    const article = articles[i];
+    const div = document.createElement("div");
+    div.classList = ["col-3"];
```

Comment:
`Element.classList` is a read-only property (at least by the spec). It seems that browsers do allow setting it, but I wouldn't rely on it.
It should be either `div.className = "col-3"` or `div.classList.add("col-3")`.

Reply:
Thanks! Have updated it